keyboards/clueboard/2x1800/config.h

```diff
@@ -69,6 +69,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
+#define RGB_DI_PIN C6
```

Comment:
Ah, awesome then.  

However, since it's a core keyboard file that's being edited here, I'd like to have @skullydazed check off on this first, since it's his board.